tools/syz-env

```diff
@@ -62,8 +75,15 @@ if [ ! "$(docker info -f "{{println .SecurityOptions}}" | grep rootless)" ]; the
 	DOCKERARGS+=" --user $(id -u ${USER}):$(id -g ${USER})"
 fi
 
-# Update docker image
-docker pull -q gcr.io/syzkaller/${IMAGE}
+
+# Build or update docker image
+if [ ! -z "$SYZ_ENV_BUILD" ]; then
+	IMAGE_NAME="syz-$IMAGE"
+	docker build "tools/docker/$IMAGE" --tag "$IMAGE_NAME" ${BUILDARGS[@]}
```

Comment:
Let's make the path relative to `$SCRIPT_DIR` -- it should be more reliable.

Reply:
@a-nogikh Agreed, done!